account_mtd/models/mtd_construct_display_message.py

```diff
@@ -18,13 +18,24 @@ def construct_error_message_to_display(self, url=None, code=None, response_token
         if 'error' in response_token.keys():
             resp_error = response_token['error']
         resp_error_message = ""
+        if 'warnings' in response_token.keys():
+            resp_error = response_token['warnings']
+            for warning in resp_error:
+                error_code = warning["code"]
+                error_message = warning["message"]
+                if 'headers' in warning.keys():
+                    resp_error_message += "\n\nError Code: \n{}\nHeaders:\n{}\nError Message: \n{}".format(error_code, warning['headers'], error_message.encode('utf-8'))
+                else:
+                    resp_error_message += "\n\nError Code: \n{}\nError Message: \n{}".format(error_code, error_message.encode('utf-8'))
```

Comment:
This can be cleaner by using the .get() fallback

```
response_errors = []

for warning in in response_token.get("warnings", []):
    response_errors.append(
        "Error Code: \n{error_code}: {header_title}{header_message}\nError Message: {error_message}".format(
            error_code=warning.get("code"),
            header_title="Headers: " if warning.get("headers") else "",
            header_message="warning.get("headers", ""),
            error_message=warning.get("message").encode("utf-8")
    )
```

Then when you need the list of headers:
```
"\n".join(response_errors)

Reply:
I've missed a few newlines out of that, but you get the idea